libs/auth/src/lockout.ts

```diff
@@ -0,0 +1,61 @@
+import { Optional } from '@votingworks/basics';
+
+const DEFAULT_NUM_INCORRECT_PIN_ATTEMPTS_ALLOWED_BEFORE_CARD_LOCKOUT = 5;
+const DEFAULT_STARTING_CARD_LOCKOUT_DURATION_SECONDS = 15;
```

Comment:
@benadida Any thoughts on these defaults? I know we've already discussed the number of incorrect PIN attempts with lockout, but we hadn't yet discussed the starting lockout duration (that will of course double with every subsequent incorrect PIN attempt)

Reply:
lgtm